packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -327,55 +331,44 @@ describe('account', () => {
-                            supply: expect.any(String),
+                        accountSpace: 80n,
+                        accountType: 'initialized',
+                        address: 'AiZExP8mK4RxDozh4r57knvqSZgkz86HrzPAMx61XMqU',
+                        authority: {
+                            address: '3xxDCjN8s6MgNHwdRExRLa6gHmmRTWPnUdzkbKfEgNAe',
                         },
-                        meta: {
-                            program: 'spl-token',
-                            space: 82n,
-                            type: 'mint',
+                        blockhash: expect.any(String),
+                        feeCalculator: {
+                            lamportsPerSignature: expect.any(String),
                         },
+                        lamports: expect.any(BigInt),
+                        programName: 'nonce',
                     },
                 },
             });
         });
-        it('can get a token account', async () => {
+        it('can get an address lookup table account', async () => {
             expect.assertions(1);
-            // See scripts/fixtures/spl-token-token-account.json
+            // See scripts/fixtures/address-lookup-table-account.json
             const variableValues = {
-                address: 'AyGCwnwxQMCqaU4ixReHt8h5W4dwmxU7eM3BEQBdWVca',
+                address: '2JPQuT3dHtPjrdcbUQyrrT4XYRYaWpWfmAJ54SUapg6n',
             };
             const source = /* GraphQL */ `
                 query testQuery($address: String!) {
                     account(address: $address) {
-                        ... on TokenAccount {
-                            data {
-                                isNative
-                                mint {
-                                    address
-                                }
-                                owner {
-                                    address
-                                }
-                                state
-                                tokenAmount {
-                                    amount
-                                    decimals
-                                    uiAmount
-                                    uiAmountString
-                                }
-                            }
-                            meta {
-                                program
-                                space
-                                type
+                        address
+                        lamports
+                        ... on LookupTableAccount {
+                            addresses
+                            authority {
+                                address
                             }
+                            deactivationSlot
+                            lastExtendedSlot
+                            lastExtendedSlotStartIndex
+                            programName
+                            accountSpace
+                            accountType
```

Comment:
I think we should eliminate this. We can completely replace this with the canonical `__type` right?

Reply:
I thought `__type` was to return the schema of the current type being queried. This would be the `type` field that comes back from a JSON parsed account, ie:

```javascript
const {
  data: {
    parsed: {
      info: {
        /* data */
      }
      type: myAccountType
    }
  }
} = await rpc.getAccountInfo(/* jsonParsed /*).send();
```

Reply:
I think I see what you meant. I took a stab at removing some unnecessary fields from the schema, including:
- `encoding`
- `accountType`